pyproject.toml

```diff
@@ -47,7 +47,6 @@ ignore_missing_imports = true
 [tool.coverage.run]
 omit = [
     "openfe/due.py",
-    "*/tests/dev/*py",
```

Comment:
Re: coverage for tests - yes, this is something Swenson enforced early on in the project's history, we should be doing everywhere imho.

In this particular case, I don't think it should be in the coverage count, because this is a purely "devtool" thing, that will never get tested / covered.

i.e. someone saying "hey I'm going to add tests for this file", wouldn't be something that I'd ever consider to be a good use of time.